code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1230,7 +1230,7 @@
 	result_amount = 3
 
 /datum/chemical_reaction/drinks/nuka_cola
-	name = "Nuka Cola"
+	name = "Nuclear Cola"

```

Comment:
>Let's not be those guys who remove the fun drinks, please.

I don't think that implied a thumbs up, unless you are referring to a discussion outside of Github.